Add FOSUserBundle.

justrau opened the pull request:
Login works through form or Facebook at /app_dev.php/login

Commit:
Add FOSUserBundle. Login works through form or Facebook.

lukasvtechceo commented on app/config/config.yml:
```diff
@@ -86,8 +87,9 @@ hwi_oauth:
-            client_id:           552137511592276
-            client_secret:       bb3b56f1cc59d2f60a3e6cb672471b8d
+            client_id:           658242424287528
+            client_secret:      58c24107ee65000502a0f65b5252651c
```
just to let you know, putting passwords and etc into git is bad practice ;)

justrau replied:
Thought so too. What would be the best practice?
%client_id%, %client_secret% defined in parameters.yml?

lukasvtechceo replied:
Yes, parameters should be added on deploy. 
You can use it with conjunction of parameters.yml.dist.

Commit:
Register and login with email. Username fields are populated with email provided during registration.
Add First name and Last name to /profile and /profile/edit.
Move facebook_client_id and facebook_client_secret out of parameters.yml.
